File: tutorials/csharp/rado/rado_i.cs
public static class radoConst
{
	//defines for service RAdo
	public const uint sidRAdo = (SocketProAdapter.BaseServiceID.sidReserved + 4);

	public const ushort idGetDataSetRAdo = ((ushort)SocketProAdapter.tagBaseRequestID.idReservedTwo + 1);
	public const ushort idGetDataTableRAdo = (idGetDataSetRAdo + 1);
}

File: tutorials/vbnet/rado/rado_i.vb
Public NotInheritable Class radoConst

	Private Sub New()
	End Sub

	'defines for service RAdo
	Public Const sidRAdo As UInteger = (SocketProAdapter.BaseServiceID.sidReserved + 4)

	Public Const idGetDataSetRAdo As UShort = (CUShort(SocketProAdapter.tagBaseRequestID.idReservedTwo) + 1)
	Public Const idGetDataTableRAdo As UShort = (idGetDataSetRAdo + 1)
End Class